### PR TITLE
drive_mirror_negative_tests: Setup iscsi target on localhost.

### PR DESCRIPTION
**Pull request by vivianQizhu**
Setup iscsi target on localhost for negative tests.

id: 1415588
Signed-off-by: Qianqian Zhu <qizhu@redhat.com>

### CODE REVIEW

**Comment by vivianQizhu** [comment]
@CongLi @yanan-fu Would you help review? Thanks a lot.

**Comment by CongLi** [comment]
Acked-by: Cong Li <coli@redhat.com>

06:00:00 INFO | PASS 1-smp_8.16384m.repeat1.run_test.Host_RHEL.m7.u4.raw.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.drive_mirror_negative_tests.cancel_blocked_jobs.on_iscsi.2raw

File: qemu/tests/cfg/drive_mirror_negative_tests.cfg [hunk 1]
```diff
@@ -53,13 +53,16 @@
             start_firewall_cmd = "iptables -A INPUT -p tcp -d 127.0.0.1 --dport 2049 -j DROP"
         - on_iscsi:
             image_format_target = raw
+            only raw
+            only 2raw
```
Comment (yanan-fu) [inline qemu/tests/cfg/drive_mirror_negative_tests.cfg]: -2raw define "image_format_target = "raw"" too,  so no need to add "image_format_target = raw" again when set "only 2raw".

**Comment by vivianQizhu** [comment]
@yanan-fu Thanks, updated.

**Comment by yanan-fu** [comment]
Acked-by: Yanan Fu yfu@redhat.com

**Comment by CongLi** [comment]
Thanks all.

**Pull request merged**